CONTRIBUTING.md

```diff
@@ -23,18 +23,34 @@ If in doubt speak to [Alan Cruikshanks](https://github.com/cruikshanks) or [Ben
 
 Create a branch with a brief but descriptive name e.g. `add-more-details-for-contributors`.
 
-Start committing your changes and **push** straight away rather than when finished.
+Start committing your changes and **push** straight away rather than when finished. We don't want your work to get lost!
 
-When you are ready for feedback create the pull request and announce your PR in our [slack channel](https://defra-digital.slack.com/messages/development/) in **Slack**.
+When you are ready for feedback create the pull request. Take the time to give it a meaningful title and detailed description as this will help those reviewing it understand what you've done.
+
+> Pro tip! If you create the PR straight after your first commit, GitHub will automatically populate the title and description for the new PR from the commit message.
+
+Also assign yourself to the PR so folks know who created it and consider adding a label to the PR. These can help potential reviewers quickly see the nature of the it.
+
+When you are ready request a review from [DEFRA/sds-group](https://github.com/orgs/DEFRA/teams/sds-group).
+
+Finally announce your PR in our [slack channel](https://defra-digital.slack.com/messages/development/) in **Slack** to make sure no one can say they didn't know about it 😀.
 
 ## Getting feedback
 
 Feedback will be provided within the PR. Decisions will be documented and reasons given.
 
 We want to encourage all to contribute and suggest ways we can improve. Anyone providing feedback should be positive and professional with any comments made about your contribution, but let us know if you feel we failed.
 
+## Approval
+
+Those that review your contribution when happy with it will leave a commit indicating so e.g. 👍, 🚢, LGTM.
```

Comment:
Not sure about this bit myself. Ideally I'd like to see everyone reviewing use the review and approve tools built into GitHub. However I thought contributors might get confused by whats an *official* approval and whats just someone saying they are happy with it.

Hence I opted for this process, but others may have a better idea, or tell me I worrying about nothing 😁

Reply:
Should this be "leave a comment" rather than "leave a commit"? Comments sound good to me.